File: Programming EF Code First/chapter5/Chapter 5 VB/Fluent/Model/Trip.vb
Namespace Model
  Public Class Trip
	Public Property Identifier() As Guid
	Public Property StartDate() As Date
	Public Property EndDate() As Date
	Public Property CostUSD() As Decimal
	Public Property RowVersion() As Byte()

	Public Property Activities() As List(Of Activity)
  End Class
End Namespace

File: Programming EF Code First/chapter3/Chapter3/BAGA_FluentAPI/Model/Trip.cs
using System;

namespace Model
{
  public class Trip
  {
    public Guid Identifier { get; set; }
    public DateTime StartDate { get; set; }
    public DateTime EndDate { get; set; }
    public decimal CostUSD { get; set; }
    public byte[] RowVersion { get; set; }
  }
}